config/locales/en/course/admin/admin.yml

```diff
@@ -4,6 +4,9 @@ en:
       admin:
         index:
           header: 'Settings'
+          publish_label: 'Publish Course'
+          publish_hint: 'Published course will be shown in the public course list'
```

Comment:
`Published courses will show up in the list of courses publicly viewable by all users.`